pkgs/os-specific/darwin/apple-sdk-12.3/CLTools_macOSNMOS_SDK.nix

```diff
@@ -0,0 +1,29 @@
+{
+  cpio,
+  fetchurl,
+  lib,
+  pbzx,
+  stdenvNoCC,
+  version,
+}:
+
+let
+  releases = builtins.fromJSON (builtins.readFile ./apple-sdk-releases.json);
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  inherit version;
+  pname = "CLTools_macOSNMOS_SDK";
+  src = fetchurl releases.${version}.${finalAttrs.pname};
+  dontBuild = true;
+  darwinDontCodeSign = true;
+  nativeBuildInputs = [
+    cpio
+    pbzx
+  ];
+  unpackPhase = ''
+    pbzx $src | cpio -idm
+  '';
+  installPhase = ''
```

Comment:
```suggestion
  postInstall = ''
```

Reply:
I’m switching this and `CLTools_Executables.nix` to `buildCommand` since they’re so trivial.